web/api/src/services/cachedProfiles/cachedProfiles.ts

```diff
@@ -54,5 +54,9 @@ export const parseChallengeStatus = (status: number): StatusEnum =>
   STATUS_ENUMS[status]
 
 export const CachedProfile = {
-  status: (profile): ProfileStatus => 'submitted_via_notary',
+  // TODO: duplicate logic from profile.cairo#_get_current_status
```

Comment:
in general I try to comment *in both places* whenever something is duplicated or needs to be kept up to date in two places.

it might be a little ugly to have comments about our javascript code within the cairo contract, but may be worth it — what do you think?

Reply:
I'd be fine including that in the Cairo code! I thought about the same thing, but I'm not sure to what degree we want the Cairo contract to "know about" the frontend (which after all may one day just be one of many consumers)?